### PR TITLE
update parent field

### PR DESCRIPTION
**Pull request by FynnBe**
still missing tests

### CODE REVIEW

File: bioimageio/spec/model/v0_4/schema.py [hunk 1]
```diff
@@ -479,11 +479,9 @@ def get_min_shape(t) -> numpy.ndarray:
...
     )
 
-    parent = fields.Nested(
-        ModelParent(),
-        bioimageio_description="Parent model from which the trained weights of this model have been derived, e.g. by "
-        "finetuning the weights of this model on a different dataset. For format changes of the same trained model "
-        "checkpoint, see `weights`.",
+    parent = fields.Union(
+        [fields.BioImageIO_ID(resource_type="model"), fields.URI(), fields.RelativeLocalPath()],
+        bioimageio_description="BioImage.IO model id or URL or local relative path to a model RDF",
```
Comment (constantinpape) [inline bioimageio/spec/model/v0_4/schema.py:484]: Please extend the description, it should be clear what a parent is, i.e. the model from which this model is derived, e.g. by fine-tuning the weights.

Comment (FynnBe) [reply bioimageio/spec/model/v0_4/schema.py]: thanks, I have included that

**Comment by FynnBe** [comment]
closes #407

**Review by constantinpape — APPROVED** [review]
LGTM

**Pull request merged**